drivers/volume/portworx/portworx.go

```diff
@@ -1293,6 +1826,58 @@ func getCloudSnapStatusString(status *api.CloudBackupStatus) string {
 	return statusStr
 }
 
+// updateRunningCommandPodListInSnap updates the snapshot annotation to track pods which might have a
+// running command
+func updateRunningCommandPodListInSnap(snap *crdv1.VolumeSnapshot, pods []v1.Pod, taskID string) error {
```

Comment:
Not specific to portworx, move to a common place

Reply:
moved